packages/gatsby-tinacms-mdx/src/useMdxForm.tsx

```diff
@@ -38,7 +38,7 @@ const matter = require('gray-matter')
 
 export function useMdxForm(
   _mdx: MdxNode | null | undefined,
-  formOverrrides: Partial<FormOptions<any>> = {}
+  formOverrides: Partial<FormOptions<any>> = {}
```

Comment:
No real changes here other than fixing this typo consistently >.>;

Reply:
I think a pirate wrote this initially